webpack/assets/javascripts/react_app/components/SearchBar/index.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../AutoCompleteSearch/AutoCompleteActions';
+import reducer from '../AutoCompleteSearch/AutoCompleteReducer';
+import { menuSelector } from '../AutoCompleteSearch/AutoCompleteSelector';
+import SearchBar from './SearchBar';
+
+
+const mapStateToProps = state => ({
+  autocomplete: {
+    ...state.autocomplete,
```

Comment:
Could you destruct individual props? It might be dungarees to pass all the state with no control over it.